R/model.R

```diff
@@ -4,24 +4,47 @@
 #' model components and runs the malaria simulation. This currently returns a
 #' dataframe with the number of individuals in each state at each timestep
 #'
+#' @param country country simulation is being run for
+#' @param timesteps the number of timesteps to run the simulation for.
 #' @param pop population. See [squire::get_population]
 #' @param parameters parameters list.
 #'   See [squire::parameters_explicit_SEEIR]
+#' @param contact_matrix_set contact_matrix_set
+#' @param max_age maximum age defaults to 100
 #' @export
-run_simulation <- function(pop, parameters) {
+run_simulation <- function(country, timesteps = NULL, pop, parameters = NULL,
```

Comment:
country, timesteps, contact_matrix_set are redundant and should be removed. They are supplied in parameters, this can be quite confusing.

Reply:
I went a bit backwards and forwards on that issue.